ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -331,7 +361,29 @@ function getValues(pendingApproval, t, actions, history) {
     ],
     cancelText: t('cancel'),
     submitText: t('approveButtonText'),
+    loadingText: t('addingCustomNetwork'),
     onSubmit: async () => {
+      let endpointChainId;
+      try {
+        endpointChainId = await jsonRpcRequest(customRpcUrl, 'eth_chainId');
+      } catch (_err) {
```

Comment:
Nit: we should at least confirm that this is a network error, and throw otherwise. That would help ensure that we don't swallow errors in our code.